### ISSUE TITLE
Resolution does not change.

### ISSUE THREAD

**Issue by AndreyRGW** [wcde/sd-webui-kohya-hiresfix#1]
![image](https://github.com/wcde/sd-webui-kohya-hiresfix/assets/50556416/d097eeda-a2c1-437f-8601-ee639c25e930)
![image](https://github.com/wcde/sd-webui-kohya-hiresfix/assets/50556416/65317704-8d29-43e1-aa91-c3221d5f51af)

Resolution after generation with this extension:

![image](https://github.com/wcde/sd-webui-kohya-hiresfix/assets/50556416/03cd4b4e-60da-4a25-a03d-3f1ee775035d)

**Comment by princessGasmask** [comment]
Yes, the final resolution will be whatever you set the sliders to.

**Comment by cvar66** [comment]
> Yes, the final resolution will be whatever you set the sliders to.

What is the purpose of the scale slider?

**Comment by princessGasmask** [comment]
HiRes Fix typically generates at a smaller resolution then upscales to the final resolution you set the sliders to. So if you have the scale set to 2, then it generates at half the size then upscales it.

**Comment by 2blackbar** [comment]
are you planning to do support for img2img ?

**Comment by AIrtistry** [comment]
Dont understand who works this extension, when I using it and I disable high res fix it's do nothing and when I combine it with high res fix normal I dont notice any changes

**Comment by cseti007** [comment]
> Dont understand how works this extension, when I using it and I disable high res fix it's do nothing and when I combine it with high res fix normal I dont notice any changes

it won't upscale for you like the original hires.fix. you have to set the desired resolution with the width and hight parameters

**Comment by AIrtistry** [comment]
> > Dont understand how works this extension, when I using it and I disable high res fix it's do nothing and when I combine it with high res fix normal I dont notice any changes
> 
> it won't upscale for you like the original hires.fix. you have to set the desired resolution with the width and hight parameters

Ok thanks, I changed the parameters to x2 of my width and heigh, activate kohya hirefix but honeslty still dont see much difference :/

**Comment by BlankFX1** [comment]
> Ok thanks, I changed the parameters to x2 of my width and heigh, activate kohya hirefix but honeslty still dont see much difference :/
Same here, with the difference that it appears to do nothing at all as I'm setting it to 2560x1440 (from usually 1536x1024) and it just fills up my VRAM as it would without this Extension and it'll just start but working at like 1 week per Iteration.
